sdks/go/pkg/beam/core/runtime/xlangx/translate.go

```diff
@@ -89,27 +89,28 @@ func PurgeOutputInput(edges []*graph.MultiEdge, p *pipepb.Pipeline) {
 	}
 }
 
+// TODO(pskevin): handle sourceInput and sinkOutput
 func VerifyNamedOutputs(ext *graph.ExternalTransform) {
```

Comment:
Just a note, that VerifyNamedOutputs and ResolveOutputIsBounded are excellent candidates for unit tests.